esrally/client/synchronous.py

```diff
@@ -172,9 +180,12 @@ def perform_request(
         # Converts all parts of a Accept/Content-Type headers
         # from application/X -> application/vnd.elasticsearch+X
         # see https://github.com/elastic/elasticsearch/issues/51816
-        if self.distribution_version is not None and self.distribution_version >= versions.Version.from_string("8.0.0"):
-            _mimetype_header_to_compat("Accept", request_headers)
-            _mimetype_header_to_compat("Content-Type", request_headers)
+        if not self.is_serverless:
+            if self.distribution_version is not None and (
```

Comment:
Same as above

Reply:
Addressed in https://github.com/elastic/rally/commit/5f2db54f79ae9dc1b5c2bf3882319268a0945c65